projects/router/test/SectaSwapSmartRouter.test.ts

```diff
@@ -0,0 +1,15 @@
+import { ether, time, constants, BN, expectRevert, expectEvent, balance } from "@openzeppelin/test-helpers";
+import { advanceBlock, advanceBlockTo } from "@openzeppelin/test-helpers/src/time";
+import { artifacts, contract, ethers } from "hardhat";
+import { parseEther } from "ethers/lib/utils";
+import { BigNumber } from "ethers";
+import { assert, expect } from "chai";
+
+
+const WrappedBNB = artifacts.require("exchange-protocol/contracts/libraries/WBNB.sol");
+const SmartRouter = artifacts.require("./SmartRouter.sol");
+
+
+
+contract("SmartRouter", ([admin, bob, carol]) => {
+});
```

Comment:
The old tests had some stableswap components, but these contracts were changed (if just removing v2/stableswap components) - there should be test coverage